rpcs3/main.cpp

```diff
@@ -497,16 +499,12 @@ int main(int argc, char** argv)
 		// Propagate command line arguments
 		std::vector<std::string> argv;
 
-		if (args.length() > 1)
+		// we skip the first entry (which should be the game)
+		for (int i = 1; i < args.length(); i++)
 		{
-			argv.emplace_back();
```

Comment:
why did you remove this?
I'd assume that argv should still be the same.
Anyway, this whole change below your comment seems to change nothing else